src/RotationSpline.cc

```diff
@@ -72,9 +72,9 @@ Quaterniond RotationSpline::Interpolate(const unsigned int _fromIndex,
   }
 
   // Fast special cases
-  if (math::equal(_t, 0.0))
+  if (equal(_t, 0.0))
```

Comment:
Here too, this could be confused with `std::equal`, so it may be worth keeping for readability.

Reply:
```suggestion
  if (math::equal(_t, 0.0))
```